src/pages/Swap.tsx

```diff
@@ -578,7 +583,7 @@ function Swap(): ReactElement {
             ? "0"
             : formatUnits(
                 formState.to.value,
-                TOKENS_MAP[formState.to.symbol].decimals,
+                tokensMap[formState.to.symbol]?.decimals ?? 18,
```

Comment:
same as above, prefer not to make up numbers.

Reply:
wasnt quiet able to figure this one out.